shared/team-building/container.tsx

```diff
@@ -638,7 +638,7 @@ class StateWrapperForTeamBuilding extends React.Component<RealOwnProps, LocalSta
         highlightedIndex={this.state.highlightedIndex}
         changeShowRolePicker={this.changeShowRolePicker}
         showRolePicker={this.state.showRolePicker}
-        showServiceResultCount={this.state.searchString !== ''}
+        showServiceResultCount={false}
```

Comment:
Should we rip out all of the result count code if we never want to show counts anymore? 

I think it'd make a lot of sense to remove it in this PR if so, but I'm okay if you want to punt that to a separate PR and just leave a TODO here.

Reply:
I'm in favor of punting because there's a chance we'll want to resurrect result counts while iterating on the design. Just created PICNIC-368 to remove it later.